vim25/soap/client.go

```diff
@@ -927,3 +930,16 @@ func (c *Client) DownloadFile(ctx context.Context, file string, u *url.URL, para
 
 	return c.WriteFile(ctx, file, rc, contentLength, param.Progress, param.Writer)
 }
+
+// execName gets the name of the executable for the current process
+func execName() string {
+	name, err := os.Executable()
+	if err != nil {
+		return "N/A"
+	}
+	_, file := filepath.Split(name)
```

Comment:
could simplify this with:
```go
return strings.TrimSuffix(filepath.Base(name), ".exe")
```
And inline string (rather than `const`) is fine here too.

Reply:
Thanks Doug! Yes code is much easier to read. Thank you.